resources/public/api/conf/1/schemas/matched-individual-resources.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Matched Individual",
+  "type": "object",
+  "properties": {
+    "_links": {
+      "type": "object",
+      "properties": {
+        "individual": {
+          "type": "object",
+          "description": "The URI for the resource to get the individual's available information.",
+          "properties": {
+            "href": {
+              "type": "string",
+              "example": "/individuals/matching/57072660-1df9-4aeb-b4ea-cd2d7f96e43"
```

Comment:
should be 57072660-1df9-4aeb-b4ea-cd2d7f96e430 for consistency

Reply:
doh! 👍